### ISSUE TITLE
MySQL 5.7.9 root password reset bug

### ISSUE THREAD

**Issue by GammaC0de** [Optware/Optware-ng#232]
Hi again 😉 
The current Optware-ng MySQL version (5.7.9) has a bug that prevents you from resetting the rooot password (see https://bugs.mysql.com/bug.php?id=79027).

Please update,
Thanks.

**Comment by alllexx88** [comment]
Hi,
I remember I had some runtime issues when I tried upgrading to 5.7.10, so I'm not willing to upgrade mysql to current as of yet. I probably will at some point in the future, but not now, as I'm lacking free time to do it properly. Instead, I found the [upstream commit that fixes the bug](https://github.com/mysql/mysql-server/commit/be8ed41ef59bd3f35ce3896190b6cf63bc058ad7) and [applied it to 5.7.9](https://github.com/Optware/Optware-ng/commit/2d8108b6fea8e60fa3e4bd6fe56c8a6d5b4f6ecd). So `5.7.9-9` version should not have this bug
